Packages/io.chainsafe.web3-unity.marketplace/Editor/AddMarketplaceScriptingDefineSymbol.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEditor;
+using System.IO;
+using UnityEngine;
+
+[InitializeOnLoad]
+public class AddMarketplaceScriptingDefineSymbol
+{
+    private const string DefineSymbol = "MARKETPLACE_AVAILABLE";
+    private static bool initialized;
+
+    static AddMarketplaceScriptingDefineSymbol()
+    {
+        ScriptingDefineSymbols.TryAddDefineSymbol(DefineSymbol);
```

Comment:
Do we still need this? Do we have any marketplace scripts in core pkg anymore? if so can't we use a combination of `ISample` and service adapters? same as ramp

Reply:
dont think so, it was for conditional for the sample scene display.

Reply:
yeah I got that we can still use a combo of the two like we did for Ramp samples

Reply:
Marketplace sample scene is a standalone scene of it's own now that we have the ability to have a single scene system. Same should be done for MUD imo.